src/Maxmind/Bundle/GeoipBundle/Command/LoadDataCommand.php

```diff
@@ -46,23 +44,33 @@ protected function configure()
     protected function execute(InputInterface $input, OutputInterface $output)
     {
         $source = $input->getArgument('source');
-
-        $bundlePath = $this->getApplication()->getKernel()->getBundle('MaxmindGeoipBundle')->getPath();
+        /** @var \Symfony\Bundle\FrameworkBundle\Console\Application $application */
```

Comment:
Yes but you get this information with the documentation about "$this->getApplication();". If you wish to reused the '$application' variable why not, but here it's not a good idea.

"GOOD CODE IS LIKE A GOOD JOKE, IT NEEDS NO EXPLANATIONS" ;)